HIP/hip-0000-hts-precompile.md

```diff
@@ -0,0 +1,299 @@
+- hip: <HIP number (this is determined by the HIP editor)>
```

Comment:
This HIP will be assigned number `206`, please rename the file to `hip-206.md`

Reply:
done